public/request/achievement/update-video.php

```diff
@@ -1,19 +1,42 @@
 <?php
 
+use App\Community\Enums\ArticleType;
+use App\Platform\Enums\AchievementType;
+use App\Platform\Models\Achievement;
 use App\Site\Enums\Permissions;
 use Illuminate\Support\Arr;
 use Illuminate\Support\Facades\Validator;
 
-if (!authenticateFromCookie($user, $permissions, $userDetails, Permissions::Developer)) {
+if (!authenticateFromCookie($user, $permissions, $userDetails, Permissions::JuniorDeveloper)) {
     abort(401);
 }
 
 $input = Validator::validate(Arr::wrap(request()->post()), [
-    'achievement' => 'required|integer',
+    'achievement' => 'required|integer|exists:mysql_legacy.Achievements,ID',
```

Comment:
```suggestion
    'achievement' => 'required|integer|exists:mysql.Achievements,ID',
```



ℹ️ The `mysql_legacy` db connection has been replaced by the default `mysql` connection.
When trying to embed a video it will result in `Database connection [mysql_legacy] not configured`.

Reply:
Thanks, I would've missed this. Resolved in https://github.com/RetroAchievements/RAWeb/pull/1526/commits/f1c62c77d3af1025004dcdb0a0634c7de7c8f5ef.